TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Persons/Index.cshtml

```diff
@@ -3,3 +3,95 @@
 @{
     ViewBag.Title = "All records";
 }
+
+<h1 class="govuk-heading-l">@ViewBag.Title</h1>
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds-from-desktop">
+        <div class="govuk-!-margin-bottom-6">
+            <form trs-action="l => l.Persons()" data-testid="search-form">
+                <div class="moj-search trs-search">
+                    <govuk-input asp-for="Search"
+                                 input-class="moj-search__input"
+                                 label-class="moj-search__label govuk-!-font-weight-bold"
+                                 id="search"
+                                 name="search"
```

Comment:
Not sure we need `id` and `name` attributes.

Reply:
Nope .. it seems to generate these anyway (albeit as "Search" with a capital)

Reply:
Removed